root/static/scripts/tests/relationship-editor.js

```diff
@@ -665,6 +665,55 @@ test('splitRelationshipByAttributes', function (t) {
     'second relationship contains drums',
   );
 
+  // This test ensures tasks are treated the same as instruments and vocals.
+
+  const task = {
+    text_value: 'stuffy stuff',
```

Comment:
Can it be given a real example value for clarity? `'dancer'`?

Reply:
Booooring. Sure.

Reply:
`bird-disguised dancer`?